lib/alexa.js

```diff
@@ -242,24 +242,22 @@ function EmitEvent() {
  * @return {undefined}
  */
 function RegisterHandlers(...args) {
-  for (let arg = 0; arg < args.length; arg += 1) {
-    const handlerObject = args[arg];
-
+  args.forEach((handlerObject) => {
```

Comment:
I don't think I agree with you on this one. I think `const handlerObject = arg` is unnecessary. We can just directly use the argument inside the arrow function, and as we're not changing that parameter at all, that's all fine. 

On error throwing, if you look at the original code, they also chose to show the index of the parameter which wasn't an object, rather than the whole object, because the whole object can be huge. That's why I used `args.indexOf(handlerObject)`.